client.js

```diff
@@ -47,7 +47,7 @@ function onError(dfd, error){
 
 function onMessage(dfd, frames) {
   var msg = Message.parse(frames);
-  log.debug("received message %s", msg);
+  log.debug(msg, "Received message from %s with id %s to %j with status %s", msg.identity, msg.rid, msg.address, msg.status);
```

Comment:
This is in the client than doesnt make sense to add client information (identity) on logging because is already logged in is file. Also we should use info log level instead.

```
log.info(msg, "Received message with id %s from %j with status %s", msg.rid, msg.address, msg.status);
```